pages/serverless-containers/concepts.mdx

```diff
@@ -87,11 +89,17 @@ Unit used to measure the resource consumption of a container. It reflects the am
 
 gRPC is supported on Serverless Containers, as long as you have enabled http2 (`h2c`) protocol.
 
-## Healthcheck
+## Health check
 
-To determine the status of a container, the default healthcheck automatically checks if basic requirements are met, to define the status as `ready`.
+To determine the status of a container, the default health check automatically checks if basic requirements are met, to define the status as `ready`. You can customize the following elements to better fit your use case:
+- The **probe type** defines the type of check to perform to define if your container is ready:
+    - The **TCP** probe will check if the TCP connection of your Serverless Container is opened on the specified port
+    - THe **HTTP** probe allows you to define a custom path to your Serverless Container to get the readiness status. It can be useful to check if all requirements are met, such as the connection to database, etc. 
```

Comment:
```suggestion
    - The **HTTP** probe allows you to define a custom path to your Serverless Container to get the readiness status. It can be useful to check if all requirements are met, such as the connection to database, etc. 
```